src/config.ts

```diff
@@ -48,10 +48,8 @@ export const configSchema = joi
           labels: ['priority'],
           message: mls`
           | Hey <author> :wave:,
-          | 
-          | Thanks for backing our project. We will handle your issue
-          | with priority support. To make sure we don't forget how special
-          | you are, we added a ${'`priority`'} label to your issue.
+          |
+          | Thanks for backing our project. We will handle your issue with priority support. To make sure we don't forget how special you are, we added a ${'`priority`'} label to your issue.
```

Comment:
@maticzav Notice this change. Line breaks were persisting in GitHub markdown and the rendering was not beautiful. Maybe this is something you should consider for your `multilines` package.

Reply:
@znarf  yeah, I noticed that but have no idea yet how to properly integrate it into `multilines`. Do you have any idea?